cmd/observe/flows.go

```diff
@@ -616,9 +616,6 @@ func getFlowsRequest(ofilter *flowFilter, allowlist []string, denylist []string)
 	if first && selectorOpts.all {
 		return nil, fmt.Errorf("cannot set both --first and --all")
 	}
-	if first && selectorOpts.follow {
-		return nil, fmt.Errorf("cannot set both --first and --follow")
-	}
```

Comment:
Note for reviewers: removing this check is fine, the Hubble server currently does [the same validation on the gRPC request object](https://github.com/cilium/cilium/blob/36237c2c5726b2158248554db849cfb6f8553b8e/pkg/hubble/observer/local_observer.go#L597-L602), and thus we'll still get an error if the server doesn't support combining `--first` and `--follow`.